environments/stg/stg.tfvars

```diff
@@ -300,80 +300,9 @@ frontends = [
   {
     name             = "idam-web-public"
     custom_domain    = "idam-web-public.aat.platform.hmcts.net"
+    mode             = "Detection"
     backend_domain   = ["firewall-prod-int-palo-aat.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-aat-platform-hmcts-net"
-    global_exclusions = [
```

Comment:
why remove these? don't you want them there so that you can tune the WAF?

Reply:
At the moment no one is available to tune because of all other concurrent works and the waf is blocking some functionality in aat

Reply:
Tuning will continue probably tomorrow